interceptor/search_attribute_translator_test.go

```diff
@@ -21,6 +19,33 @@ func TestTranslateSearchAttribute(t *testing.T) {
 
 func generateSearchAttributeObjs() []objCase {
 	return []objCase{
+		{
+			objName:     "nil",
+			containsObj: false,
+			makeType: func(name string) any {
+				return nil
+			},
+		},
+		{
+			objName:     "nil SearchAttributes",
+			containsObj: false,
+			makeType: func(name string) any {
+				return &persistence.WorkflowExecutionInfo{
+					SearchAttributes: nil,
```

Comment:
curious how the translation work in this case.

Reply:
If we see `SearchAttributes = nil`, we should just ignore it. (It would be this wf has no custom search attributes, so no translation needed)